oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/web/OAuth2AuthorizationEndpointFilterTests.java

```diff
@@ -583,11 +601,6 @@ public void doFilterWhenUserConsentRequiredAndAuthorizationRequestThenUserConsen
 
 		this.filter.doFilter(request, response, filterChain);
 
-		verifyNoInteractions(filterChain);
```

Comment:
Is there a reason these lines were removed?

Reply:
See comment about `OAuth2AuthorizationCodeGrantTests`

https://github.com/spring-projects-experimental/spring-authorization-server/pull/280#discussion_r640454894